src/test/java/HWTestForLoginTest.java

```diff
@@ -0,0 +1,66 @@
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+    public class HWTestForLoginTest {
+    @Test
+    void test1 () throws InterruptedException {
```

Comment:
testLoginSuccess name is better than test1. Test name should show what this test does.